examples/alerts/alerts.md

```diff
@@ -275,7 +275,7 @@ rules:
     message: Thanos Sidecar {{$labels.job}} {{$labels.pod}} is unhealthy for {{ $value
```

Comment:
When I checked it again since we removed `pod` aggregation, `{{$labels.pod}}` won't be available.

Reply:
@hwoarang I could be wrong here. Better to test it using the alert tests we have.